code/game/objects/items/kitchen.dm

```diff
@@ -150,8 +150,8 @@
 	item_state = "knife"
 	desc = "A military combat utility survival knife."
 	embedding = list("pain_mult" = 4, "embed_chance" = 65, "fall_chance" = 10, "ignore_throwspeed_threshold" = TRUE)
-	force = 20
-	throwforce = 20
+	force = 16
```

Comment:
```suggestion
	force = 17
```
it's still a thing that has to come through cargo

Reply:
Sure, but I figure just a +1 in comparison to the other knives would be fine, and just being under 1 force of a wielded spear seems a bit much.